README.rst

```diff
@@ -116,6 +119,52 @@ Usage
       latest     Get the latest tag in a changelog.
       tag        Create an annotated git tag based on a changelog entry.
       update     Updates the version and date of the most recent changelog entry.
+      create     Creates a new CHANGELOG.md file with a standard header.
+      entry      Add a quick entry to the current version.
+      next       Create a new unreleased version in changelog file.
+
+next
+----
+
+Create a new unreleased version in changelog file.
+
+::
+
+   Usage: chag next
+
+   Creates a new unreleased version inside of the changelog file.
+
+   Options:
+     --help     Displays this message.
+
+entry
```

Comment:
@dpsenner Vulturing PRs is generally only acceptable if the PR is not active, you can clearly see with the comments going on in here that, that is not the case and this PR is indeed active. The PR not being merged is because the maintainer did not respond to the PR for an extremely long time.

Once again, I did not say that people can't use my code, and once again, I am fully aware of what the MIT license allows. Your strawman argument falls appart when you look at the content of my comments THIS VERY YEAR, however, your extremely aggressive and obnoxious behavior is not excused by the MIT license.

The MIT license doesn't suggest you to try to get another PR closed so you can have yours merged after simply aggregating other's work and putting it together. You should familiarize yourself with collaboration etiquette as I suggested before.

Now, I would appreciate if you let me handle my PR without further interruptions or distractions unless you are explicitly contributing to things related to this PR specifically.

Thank you.